src/scripts/main.js

```diff
@@ -21,7 +21,24 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
```

Comment:
The use of `data.hasOwnProperty(key)` is appropriate here to ensure that only the object's own properties are iterated over. However, keep in mind that this method does not account for properties inherited through the prototype chain. In this context, it's not an issue since `food` is a plain object, but it's something to consider in more complex scenarios.